planner/core/casetest/testdata/expression_rewriter_suite_in.json

```diff
@@ -1,17 +1 @@
-[
-  {
-    "name": "TestMultiColInExpression",
-    "cases": [
-      "select * from t1 where (t1.a, t1.b) in (select a, b from t2)",
-      "select * from t1 where (t1.a, t1.b) not in (select a, b from t2)",
-      "select t1.a from t1 group by t1.a having (a, count(1)) in (select a, b from t2)",
-      "select t1.a from t1 group by t1.a having (a, count(1)) not in (select a, b from t2)"
-    ]
-  },
-  {
-    "name": "TestBitFuncsReturnType",
-    "cases": [
-      "select bit_or(a * b) from t"
-    ]
-  }
-]
+[]
```

Comment:
Still need this file or not?

Reply:
I'll remove all empty json files (and corresponding loading functions).